spec/visitorSpec.js

```diff
@@ -0,0 +1,20 @@
+visit = require("../src/visitor");
+var fs = require('fs');
+
+describe("spies", function () {
+  var visitor = new Visitor("bob", "smith", 2 / 1 / 2019, 7, "sally");
+  it('should spy on save', function () {
+    var spy = spyOn(visitor, 'save');
```

Comment:
The spy here is not required, it blocks you from actually running the save method because that is its purpose. SpyOn() - It replaces the spied method with a stub, and does not actually execute the real method.